app/models/manageiq/providers/ansible_tower/configuration_manager/configuration_script.rb

```diff
@@ -8,7 +8,7 @@ def run(vars = {})
   end
 
   def merge_extra_vars(external)
-    internal = variables.empty? ? {} : JSON.parse(variables)
+    internal = variables.empty? ? {} : variables
```

Comment:
`variables` are never nil - the Tower API always returns either the value or  `''`.